ports/esp32s2/ai_thinker_esp_12k/mpconfigboard.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2019 Scott Shawcroft for Adafruit Industries
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+// Micropython setup
+
+// Same setup as the Saola board but with no Neopixel on board
+
+#define MICROPY_HW_BOARD_NAME       "NodeMCU-32-S2 w/ESP-12K 08"
+#define MICROPY_HW_MCU_NAME         "ESP-12K 08"
```

Comment:
Is this just the module or is the dev board?

Reply:
A Development Board base on esp32-s2

Reply:
A Development Board base on esp32-s2

Reply:
It's this, https://docs.ai-thinker.com/en/12k_development_board_esp32-s2, is that right? Sometimes it seems to be called "ESP-12K-KIT" or "ESP-12K NodeMCU". Could you pick what seems to be the best name and change the directory name to that (e.g. `ai_thinker_esp_12k_nodemcu`), and also change the strings here and in `mpconfigboard.mk`?

Another link: https://www.tinytronics.nl/shop/en/development-boards/microcontroller-boards/with-wi-fi/ai-thinker-nodemcu-32-s2-esp-12k

I just don't want the board to be confused with the bare module.

Also, I don't see the "08" designation anywhere. I don't see where that came from. If you don't need it, let's remove it.

Reply:
ok